index.html

```diff
@@ -122,13 +122,29 @@ <h1>
           <div
             class="swiper-slide work"
             data-hash="work-1"
+            data-color="#90919c"
```

Comment:
i hope this isnt some ugly bright colour

Reply:
this is grey 😭